dali/python/nvidia/dali/external_source.py

```diff
@@ -259,6 +259,9 @@ def __init__(self, source = None, num_outputs = None, *, cycle = None, layout =
         self._cuda_stream = cuda_stream
         self._use_copy_kernel = use_copy_kernel
 
+        import nvidia.dali.ops
```

Comment:
move this import to the top of the file?

Reply:
This is a circular import, it won't work on top of the file.